lib/pci.py

```diff
@@ -39,6 +41,152 @@ def get_domains():
         return list(set(domains))
     return []
 
+def  is_nvmf():
+    """
+    Verify if nvmf is configured
+
+    :return: True if nvmf is configure, False if nvmf is not configured
+    """
+
+    try:
+        nvme_list_subsys_output_bytes = subprocess.check_output(['nvme', 'list-subsys'])
```

Comment:
you can reuse runcmd from library https://github.com/lop-devops/tests/blob/5f18b85850e53a143dedfdb2a1d29cfdca040e86/lib/helper.py#L31